nodejs/awsx/aws_test.go

```diff
@@ -230,10 +230,6 @@ func Test_Examples(t *testing.T) {
 			Dir:       path.Join(cwd, "../examples/alb/ec2"),
 			StackName: addRandomSuffix("ec2"),
 		}),
-		testBase.With(integration.ProgramTestOptions{
-			Dir:       path.Join(cwd, "../examples/alb/ec2InlineListener"),
-			StackName: addRandomSuffix("ec2"),
-		}),
```

Comment:
These tests were atually bogus.  but didn't realize this until the long tests ran.